### PR TITLE
nav: persist location state for groups and messages when navigating between tabs on mobile

### PR DESCRIPTION
**Pull request by latter-bolden**
OTT

Double clicking returns you to the root location.

https://github.com/tloncorp/landscape-apps/assets/90741358/eed83afb-5809-4a8c-a733-9f0c9e4ce41a

Fixes LAND-848

### CODE REVIEW

**Comment by urcades** [comment]
This is looking good

File: ui/src/state/ephemeral.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import create from 'zustand';
```
Comment (arthyn) [inline ui/src/state/ephemeral.ts:1]: we use "local.ts" for random stuff like this

**Review by arthyn — APPROVED** [review]
lgtm, might be good to combine that state with local

File: ui/src/state/ephemeral.ts (hunk 1 above)
Comment (latter-bolden) [reply ui/src/state/ephemeral.ts:1]: was going to add there, but looked like that still gets persisted to `localStorage`?

this felt like something that should fully reset when you close the app

Comment (latter-bolden) [reply ui/src/state/ephemeral.ts:1]: jk, missed the partialize call there. Will update

**Pull request merged**